Resources/public/js/views/services/plugins/ez-publishdraftplugin.js

```diff
@@ -99,7 +99,9 @@ YUI.add('ez-publishdraftplugin', function (Y) {
                 notificationIdentifier = this._buildNotificationIdentifier(content.get('id'));
 
             if ( error ) {
-                this._parseServerFieldsErrors(response, this._serverSideErrorCallback);
+                if (response) {
```

Comment:
you just moved the question raised in https://github.com/ezsystems/PlatformUIBundle/pull/769#discussion_r94311013 here but the question remains. a `response` is supposed to be always available here, so unless I'm missing something the fix is not to check is the response is there but to make sure it's always there so you need to fix the case(s) where the response is not provided.